scipy/stats/tests/test_multivariate.py

```diff
@@ -1930,11 +1930,10 @@ def test_logpdf_4x4(self):
 
 class TestSpecialOrthoGroup:
     def test_reproducibility(self):
-        np.random.seed(514)
-        x = special_ortho_group.rvs(3)
-        expected = np.array([[-0.99394515, -0.04527879, 0.10011432],
-                             [0.04821555, -0.99846897, 0.02711042],
-                             [0.09873351, 0.03177334, 0.99460653]])
+        x = special_ortho_group.rvs(3, random_state=np.random.default_rng(514))
```

Comment:
Oops, `Generator`s don't produce the same results as the global `RandomState` given the same seed.

Reply:
Thanks, done. It seems that the second half of the test uses legacy numpy random API (`np.random.RandomState`), so I deleted it and adjusted the stored value.